src/psyclone/dynamo0p3.py

```diff
@@ -6356,13 +6591,13 @@ class HaloReadAccess(HaloDepth):
     accessed in a particular kernel within a particular loop nest
 
     '''
-    def __init__(self, field):
+    def __init__(self, field, sym_table):
```

Comment:
doc string for `sym_table` please.

Reply:
Oops again.

Reply:
As before. Please move docstrings from __init__ to class description. Please also add a full stop to `param field`.

Reply:
Oops, oops and done. I can't have been very awake!